### PR TITLE
Add method for persisted queue stats

### PR DESCRIPTION
**Pull request by keyan**
Also see: https://github.com/edgurgel/verk_web/pull/27

This PR adds an additional method to `lib/verk/stats.ex` that returns total processed/failed job counts for every queue in the format:

```
%{<queue> => %{total_finished: <# finished>, total_failed: <# failed>}, ...}
```

### CODE REVIEW

**Comment by keyan** [comment]
I think Coveralls is a bit confused as it says a line isn't being covered, but it clearly is being executed.

File: lib/verk/stats.ex [hunk 1]
```diff
@@ -12,6 +12,17 @@ defmodule Verk.Stats do
     %{processed: to_int(processed), failed: to_int(failed)}
   end
 
+  @doc """
+  Total amount of processed and failed jobs per queue
+  """
+  @spec queue_totals(GenServer.server) :: Map.t
+  def queue_totals(redis \\ Verk.Redis) do
+    for {queue, _} <- Application.get_env(:verk, :queues), into: %{} do
```
Comment (edgurgel) [inline lib/verk/stats.ex]: There's just one problem with this approach. One can add/remove queues dynamically so the application env is not enough to give this information. 

Why don't we simplify for now receive the queue name as part of the `queue_totals` arguments and call from here:

https://github.com/edgurgel/verk_web/blob/436806978af3d37dad7ba10218b4bb084241f71b/web/views/page_view.ex#L4-L7

What do you think?

Comment (keyan) [inline lib/verk/stats.ex]: Ah that sounds much cleaner!

**Comment by keyan** [comment]
Ready for another look @edgurgel. I updated this PR as well as https://github.com/edgurgel/verk_web/pull/27 to reflect your suggestions.

**Comment by edgurgel** [comment]
LGTM!

**Comment by edgurgel** [comment]
I will release later today a new version with the changes you sent. Thank you again 👍

**Pull request merged**